xcode/Subconscious/Shared/Services/Noosphere/NoosphereService.swift

```diff
@@ -164,113 +164,110 @@ actor NoosphereService:
         }
         
         let noosphere = try noosphere()
-        logger.debug("Initializing Sphere with identity: \(identity)")
+        logger.log("Initializing Sphere with identity: \(identity)")
         let sphere = try Sphere(
             noosphere: noosphere,
             identity: identity
         )
         self._sphere = sphere
-        logger.debug("Initialized and cached Sphere with identity: \(identity)")
+        logger.log("Initialized and cached Sphere with identity: \(identity)")
         return sphere
     }
     
-    nonisolated private func spherePublisher() -> AnyPublisher<Sphere, Error> {
-        Future {
-            try await self.sphere()
-        }
-        .eraseToAnyPublisher()
-    }
-
     func identity() async throws -> Did {
-        try await self.sphere().identity()
+        try await errorLoggingService.capturing {
+            try await sphere().identity()
+        }
     }
 
     nonisolated func identityPublisher() -> AnyPublisher<Did, Error> {
-        self.spherePublisher().flatMap({ sphere in
-            sphere.identityPublisher()
-        })
+        Future {
+            try await self.identity()
+        }
         .eraseToAnyPublisher()
     }
 
     func version() async throws -> Cid {
-        try await self.sphere().version()
+        try await errorLoggingService.capturing {
+            try await sphere().version()
+        }
     }
     
     nonisolated func versionPublisher() -> AnyPublisher<Cid, Error> {
-        self.spherePublisher().flatMap({ sphere in
-            sphere.versionPublisher()
-        })
+        Future {
+            try await self.version()
+        }
```

Comment:
Use our Future helper to expose publishers. This lets us call methods from `NoosphereService`, rather than `Sphere` or `Noosphere`, ensuring we get the error logging.